consts/consts.go

```diff
@@ -38,13 +38,13 @@ var (
 			"name":           "mp3 audio",
 			"targetFormat":   "mp3",
 			"defaultBitRate": 192,
-			"command":        "ffmpeg -i %s -map 0:0 -b:a %bk -v 0 -f mp3 -",
+			"command":        "ffmpeg -i %s -map 0:0 -b:a %bk -v 0 -f %f -",
 		},
 		{
 			"name":           "opus audio",
-			"targetFormat":   "oga",
+			"targetFormat":   "opus",
```

Comment:
What was the reasoning for this renaming? `oga` (ogg audio) is the container used for opus encoded files. I don't think it would change anything but would like to understand your reasons